create-doks-with-terraform-flux/main.tf

```diff
@@ -0,0 +1,148 @@
+terraform {
+  required_version = ">=1.0.2"
+}
+
+# SSH
+locals {
+  known_hosts = "github.com ${var.github_ssh_pub_key}"
+}
+
+resource "tls_private_key" "main" {
+  algorithm = "RSA"
+  rsa_bits  = 4096
+}
+
+# DOKS
+data "digitalocean_kubernetes_cluster" "primary" {
+  name = var.doks_cluster_name
+  depends_on = [
+    digitalocean_kubernetes_cluster.primary
+  ]
+}
+
+resource "digitalocean_kubernetes_cluster" "primary" {
+  name    = var.doks_cluster_name
+  region  = var.doks_cluster_region
+  version = var.doks_cluster_version
+
+  node_pool {
+    name       = "${var.doks_cluster_name}-pool"
+    size       = var.doks_cluster_pool_size
+    auto_scale = true
+    min_nodes  = var.doks_cluster_pool_nodes.min
+    max_nodes  = var.doks_cluster_pool_nodes.max
```

Comment:
Enabling auto-scale seems unnecessary for this guide. I worry readers would potentially be confused about whether using Flux requires auto-scale.

Reply:
You're right, no need to complicate things. Fixed in the latest commit.